src/test/java/lotto/LottoTest.java

```diff
@@ -0,0 +1,72 @@
+package lotto;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+//- [ ] 로또 번호는 총 6개이다
+//- [ ] 로또 번호는 중복되지 않아야 한다
+//    - [ ] 로또 번호 중복 시 IllegalArgumentException을 발생시킨다
+
+public class LottoTest {
+
+    @Test
+    void 로또_번호_6개로_생성() {
+        List<Integer> numbers = Arrays.asList(1, 2, 3, 4, 5, 6);
+
+        Lotto lotto = new Lotto(numbers);
```

Comment:
```suggestion
        Lotto lotto = new Lotto(1, 2, 3, 4, 5, 6);
```
가변 인자를 받는 생성자도 추가하면 Lotto 생성이 더 편리해 지지 않을까?